traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -614,7 +614,7 @@ func (ds *TODeliveryService) Read(h http.Header, useIMS bool) ([]interface{}, er
 	for _, ds := range dses {
 		switch {
 		// NOTE: it's required to handle minor version cases in a descending >= manner
-		case version.Major >= 4 && version.Minor >= 1:
+		case version.Major > 4 || (version.Major == 4 && version.Minor >= 1):
```

Comment:
This is fine, though `Version.GreaterThanOrEqualTo()` can be used now, too

https://github.com/apache/trafficcontrol/blob/6c532922e706481cb944820b8fa6af20cc3d11ee/traffic_ops/traffic_ops_golang/cdn/cdns.go#L67

Reply:
Noted; though I don't want to mess with that any more; I need to rebase the DS Active flag changes every time it changes.